doc/cloudfuse_mount_list.md

```diff
@@ -37,31 +37,34 @@ cloudfuse mount list
       --block-cache-prefetch uint32       Max number of blocks to prefetch.
       --block-cache-prefetch-on-open      Start prefetching on open or wait for first read.
       --block-size-mb uint                Size (in MB) of a block to be downloaded during streaming.
+      --bucket-name string                Name of s3 bucket.
       --cache-size-mb uint32              max size in MB that file-cache can occupy on local disk for caching
       --config-file string                Configures the path for the file where the account credentials are provided. Default is config.yaml in current directory.
       --container-name string             Configures the name of the container to be mounted
       --cpk-enabled                       Enable client provided key.
       --disable-compression               Disable transport layer compression.
       --disable-version-check             To disable version check that is performed automatically
       --disable-writeback-cache           Disallow libfuse to buffer write requests if you must strictly open files in O_WRONLY or O_APPEND mode.
+      --enable-symlinks                   whether or not symlinks should be supported
+      --endpoint string                   S3 endpoint url.
       --entry-timeout uint32              The entry timeout in seconds.
       --file-cache-timeout uint32         file cache timeout (default 120)
       --foreground                        Mount the system in foreground mode. Default value false.
       --hard-limit                        File cache limits are hard limits or not.
       --high-disk-threshold uint32        percentage of cache utilization which kicks in early eviction (default 90)
       --ignore-open-flags                 Ignore unsupported open flags (APPEND, WRONLY) by cloudfuse when writeback caching is enabled. (default true)
       --ignore-sync                       Just ignore sync call and do not invalidate locally cached file.
-      --log-file-path string              Configures the path for log files. Default is $HOME/.cloudfuse/cloudfuse.log (default "$HOME/.cloudfuse/cloudfuse.log")
+      --log-file-path string              Configures the path for log files. Default is /home/jfan/.cloudfuse/cloudfuse.log (default "/home/jfan/.cloudfuse/cloudfuse.log")
```

Comment:
This looks like a bug in the documentation generation

Reply:
Fixed.